arch/sim/src/Makefile

```diff
@@ -284,7 +284,7 @@ $(COBJS) $(LINKOBJS): %$(OBJEXT): %.c
 	$(call COMPILE, $<, $@)
 
 $(HOSTOBJS) $(HEADOBJ): %$(OBJEXT): %.c
-	$(Q) echo "CC:  $<"
+	$(Q) echo $(ECHO_OPT)"CC:  $<"
```

Comment:
can we create a new ECHO macro like MAKE? I think we need replace all place which use echo an make directly

Reply:
This change just custom for cc/cxx compile line, and the ECHO_OPT consist of two parts:

Begin (escape and carriage ret) :
```bash
	$(if $(BASH),$(eval SHELL=$(BASH)))
	@echo $(ECHO_OPT)"CPP: $1->$2 "
```
Command:
```bash
	$(Q) $(CPP) $(CPPFLAGS) $($(strip $1)_CPPFLAGS) $(abspath $1) -o $(abspath $2)
```
End (escape and carriage ret) :
```bash
	$(if $(ECHO_OPT),@echo $(ECHO_OPT))
```
so it can not be set as a macro

Reply:
How about ECHO_BEGIN, ECHO_END ?